plugins/modules/keycloak_group.py

```diff
@@ -245,7 +340,7 @@ def main():
                                              ['token', 'auth_realm', 'auth_username', 'auth_password']]),
                            required_together=([['auth_realm', 'auth_username', 'auth_password']]))
 
-    result = dict(changed=False, msg='', diff={}, group='')
+    result = dict(changed=False, msg='', diff={})
```

Comment:
Why are you removing the return value `group`?

Reply:
Well, this is seemingly a little artifact I stumbled upon while working on this PR. This `group` thing does nothing and is never used, or otherwise said it always sets an additional field `group` in the return dict which is always the empty string.
My personal guess is that the original idea here was to use this for what later had become known as `end_state`, or maybe also something completely different, not sure. Fact is that currently it is just pure noise without any purpose so I removed it.

Technically one can argue that this change has nothing to do with the actual purpose of this PR and should not belong here and I'm happy to undo this change when requested. Making this its own PR seems personally a lot of effort for this tiny tidbit but when it it is the right thing to do it is the right thing to do.

Reply:
This is technically a breaking change, though a very minor one which probably nobody will notice. I would still not merge it as part of a feature request (that is backported to the current stable branch). If you don't want to I don't mind creating a PR for it, but that change definitely shouldn't be in this PR.

Reply:
Fair enough, undid this change.